mmdet/apis/train.py

```diff
@@ -96,9 +96,10 @@ def auto_scale_lr(cfg, distributed, logger):
         num_gpus = len(cfg.gpu_ids)
 
     # calculate the batch size
-    batch_size = num_gpus * cfg.data.samples_per_gpu
```

Comment:
`Training with {num_gpus} GPU(s) with {samples_per_gpu} samples per GPU. The total batch size is {batch_size}.`